views/table.html

```diff
@@ -3,7 +3,11 @@
   <h3 class="s-q-item__title">{{ item.title }}</h3>{% endif %} {%- if item.subtitle and item.subtitle !== '' %}
   <div class="s-q-item__subtitle">{{ item.subtitle }}</div>
   {%- endif %}
-  <div style="overflow-x: auto">
+  <div style="overflow-x: auto;padding: 0 1px;">
+    {%- if item.options.showTableSearch === true %}
+    <label for="search-form-input" hidden>Suchen</label>
+    <input id="search-form-input" class="s-input-field" type="search" placeholder="Bitte Suche eingeben" maxlength="20" value="" autocapitalize="off" autocomplete="off" spellcheck="false">
```

Comment:
using classes, not ids since there can be multiple tables in one article

Reply:
Didn't think about that. Good point! Should I rather put the `input` element inside the `label` tag like this?
```
<label hidden>Suchen
  <input>
</label>
```

Reply:
Sounds good to me

Reply:
I'll use the `aria-label` for this (as discussed)